front/lib/connector_providers.ts

```diff
@@ -27,7 +28,8 @@ export const CONNECTOR_CONFIGURATIONS: Record<
     isBuilt: true,
     logoPath: "/static/notion_32x32.png",
     description:
-      "Authorize granular access to your company's Notion workspace, by top-level pages.",
+      "Authorize granular access to your company's Notion workspace, by top-level pages. External files and content behind links are not indexed.",
```

Comment:
"Content indexed will be made available to your workspace regardless of authorizations in Notion" ?